src/_includes/child-links.njk

```diff
@@ -0,0 +1,15 @@
+<div class="child-links">
+    {% set navPages = collections.all | eleventyNavigation %}
+    {%- for entry in navPages %}
+        {% if page
+            .fileSlug
+            .includes(entry.title.toLowerCase()) %}
```

Comment:
I think if you use a parameter in `eleventyNavigation(page.url)` when constructing the `navPages`, you can skip this bit.  (And also that should be a little more correct in case titles and URLs diverge, I think?)

Reply:
Didn't know you could do that, I was positively surprised by this feature! Although `eleventyNavigation(page.url)` didn't seem to work for us, but `eleventyNavigation(page.fileSlug)` did the trick 🥳

Reply:
Seems to be how you build your `parent` keys, but unless you have a strong opinion that `page.url` should be used I think we're fine with `page.fileSlug` right? 😅